packages/agent/src/agent/http/index.ts

```diff
@@ -205,7 +205,10 @@ export class HttpAgent implements Agent {
     } else {
       const location = typeof window !== 'undefined' ? window.location : undefined;
       if (!location) {
-        throw new Error('Must specify a host to connect to.');
+        this._host = new URL('https://icp-api.io');
+        console.warn(
```

Comment:
@krpeacock shouldn't this warning only be thrown when agent-js is used in the browser and not in NodeJS?